source/common/http/utility.h

```diff
@@ -313,6 +313,9 @@ struct LocalReplyData {
   const absl::optional<Grpc::Status::GrpcStatus> grpc_status_;
   // Tells if this is a response to a HEAD request.
   bool is_head_request_ = false;
+  // Tells whether the localReply HTTP status code should reflect the called
+  // service's response.
```

Comment:
Can we clarify whether the the called service's response means 'comes from the response headers object'?